app/jobs/webhooks/close_pull_request_job.rb

```diff
@@ -9,7 +9,7 @@ def perform(train_id, pr_attributes)
     Train.find(train_id).open_active_prs_for(head_ref).where(number:).find_each do |pr|
       pr.update_or_insert!(pr_attributes)
       pr.release.event_stamp!(reason: :pr_merged, kind: :success, data: {url: pr.url, number: pr.number, base_branch: pr.base_ref})
-      Action.complete_release!(pr.release) if pr.post_release?
+      Action.complete_release!(pr.release) if pr.release.post_release_failed?
```

Comment:
unsure of this. the new release finalization failure notification may become too noisy if it tries and fails for every PR close (ongoing ones).